packages/liveblocks-react-comments/src/utils/url.ts

```diff
@@ -14,6 +15,8 @@ export function setQueryParams(
     }
   }
 
+  urlObject.hash = `#${hash}`;
```

Comment:
Idea: I think we should only set the hash if the incoming URL doesn't have one already, that way we respect the users' intention instead of overriding it.